core/app/services/spree/gift_cards/redeem.rb

```
@@ -0,0 +1,17 @@
+module Spree
+  module GiftCards
+    class Redeem
+      prepend Spree::ServiceModule::Base
+
+      def call(gift_card:, order:)
```

RuboCop (1.75.5)

[warning] Unused method argument - `order`.

(Lint/UnusedMethodArgument)